Description: Comply with Expat+no-false-attribs license
 https://raw.github.com/isaacs/npm/594bbbd2/LICENSE
 When npm encounters a problem of any sort, it displays a message
 that links to its own bug tracker and mailing list.
 The no-false-attribs clause of its license requires the npm debian
 package to displays its own bug tracker instead.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2012-03-13
Index: npm/lib/utils/error-handler.js
===================================================================
--- npm.orig/lib/utils/error-handler.js
+++ npm/lib/utils/error-handler.js
@@ -230,9 +230,9 @@ function errorHandler (er) {
   default:
     log.error("", er.stack || er.message || er)
     log.error("", ["If you need help, you may report this log at:"
-                  ,"    <http://github.com/isaacs/npm/issues>"
-                  ,"or email it to:"
-                  ,"    <npm-@googlegroups.com>"
+                  ,"    <http://bugs.debian.org/npm>"
+                  ,"or use"
+                  ,"    reportbug --attach "+path.resolve("npm-debug.log")+" npm"
                   ].join("\n"))
     printStack = false
     break
